docs/index.yaml

```diff
@@ -331,7 +331,8 @@ entries:
   - apiVersion: v1
     appVersion: "3.1"
     created: "2021-05-06T12:00:48.369807-07:00"
-    description: A Helm chart for installing the HPE Volume Driver for Kubernetes FlexVolume plugin
+    description: A Helm chart for installing the HPE Volume Driver for Kubernetes
```

Comment:
Why do we see changes here?

Reply:
Its probably we have different helm version and it wraps the description for some of us which udpates timestamp. We have seen in prior build as well. Its involuntary 
```
 helm version
version.BuildInfo{Version:"v3.2.4", GitCommit:"0ad800ef43d3b826f31a5ad8dfbb4fe05d143688", GitTreeState:"dirty", GoVersion:"go1.14.3"
```